.travis.yml

```diff
@@ -4,11 +4,13 @@
 language: generic
 
 os: osx
-osx_image: xcode6.4
+osx_image: xcode9.4
```

Comment:
This will be removed next re-rendering. We should discuss with @conda-forge/core the move to `9.4` ASAP.

Reply:
I am not following this comment.

Reply:
Was this changed done via conda-smithy re-render? I'm not sure if we already updated it upstream. If it was manually changed then a re-render may revert it back.

Reply:
Yes. I rerendered locally with conda-smithy. Let's tag the bot one more time to be sure.

Reply:
Then we are good.